src/api/IBMi.ts

```
@@ -1308,5 +1327,21 @@ export default class IBMi {
       runtimeCcsid: this.runtimeCcsid,
       userDefaultCCSID: this.userDefaultCCSID,
     };
-  } 
+  }
+
+  async checkUserSpecialAuthorities(authorities: ("*ALLOBJ" | "*AUDIT" | "*IOSYSCFG" | "*JOBCTL" | "*SAVSYS" | "*SECADM" | "*SERVICE" | "*SPLCTL")[], user?: string) {
```

I don't think this API should throw an error. I do think that it should instead return a valid result.

Perhaps `{valid: boolean, missing: string[]}`. I also think we need a type for the list of authorities.

---

Sounds reasonable. Let's do this.